zeroconf/__init__.py

```diff
@@ -887,13 +887,33 @@ def add_question(self, record: DNSQuestion) -> None:
         self.questions.append(record)
 
     def add_answer(self, inp: DNSIncoming, record: DNSRecord) -> None:
+
+        """Only support for unique answers"""
+        if (
+            record.type == _TYPE_TXT
+            or record.type == _TYPE_SRV
+            or record.type == _TYPE_A
+            or record.type == _TYPE_AAAA
+        ):
+            assert record.unique
+
         """Adds an answer"""
         if not record.suppressed_by(inp):
             self.add_answer_at_time(record, 0)
 
     def add_answer_at_time(self, record: Optional[DNSRecord], now: Union[float, int]) -> None:
         """Adds an answer if it does not expire by a certain time"""
         if record is not None:
+
+            """Only support for unique answers"""
+            if (
+                record.type == _TYPE_TXT
+                or record.type == _TYPE_SRV
+                or record.type == _TYPE_A
+                or record.type == _TYPE_AAAA
+            ):
+                assert record.unique
```

Comment:
I'd think about putting those `if/assert` blocks in a function, but it's no biggie. I'd at least remove the comments, the code is crystal clear on what it's going on here.

Reply:
Happy to remove comment, do you want me to resubmit a PR?

Reply:
That's up to you, we can merge as-is.

Reply:
I went ahead and merged it, we can clean it up later.

Reply:
New PR submitted t address this https://github.com/jstasiak/python-zeroconf/pull/225